src/utils/_settings.less

```diff
@@ -1,5 +1,10 @@
+// Dependencies path
 @bs-path: "../node_modules/bootstrap";
 @oui-path: "../node_modules/ovh-ui-kit";
 
+// Fonts folder
 @oui-icon-dist-folder: "./icons";
 @oui-font-source-sans-pro-folder: "./fonts/source-sans-pro";
+
+// OVH Universe
+@ovh-universe-color: @brand-primary;
```

Comment:
Does adding this variable into the `README` file might be useful?

Reply:
Yes, it can be. I'll add it